conf/etc/v2/system_health_hierarchy.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
@ajaysrivas I did not move this to a class since I think it will be huge after we add more elements to this, so keeping a separate file might be a good thing.

Reply:
Ok. What is the plan for exposing hierarchy to the caller? Using this json file?

Reply:
See that the file is used in HealthHierarchy class. So we will be exposing that class?

Reply:
I think caller needs to know the elements (to be passed as an argument in get_system_health() method), we could expose CLUSTER_ELEMENTS class.

Reply:
We need an api to return the hierarchy, can be taken up later.